tests/pipeline_backend_test.py

```diff
@@ -116,109 +118,112 @@ def _create_mock_pcollection():
     def test_values(self, mock_rrshift):
-        mock_pcollection, ops = self._test_helper()
-        ops.values(mock_pcollection, "stage_name")
+        mock_pcollection, backend = self._test_helper()
+        backend.values(mock_pcollection, "stage_name")
         mock_rrshift.assert_called_once_with("unique_label")
 
     @patch("apache_beam.transforms.ptransform.PTransform.__rrshift__")
     def test_sample_fixed_per_key(self, mock_rrshift):
-        mock_pcollection, ops = self._test_helper()
-        ops.sample_fixed_per_key(mock_pcollection, 1, "stage_name")
+        mock_pcollection, backend = self._test_helper()
+        backend.sample_fixed_per_key(mock_pcollection, 1, "stage_name")
         mock_rrshift.assert_called_once_with("unique_label")
 
     @patch("apache_beam.transforms.ptransform.PTransform.__rrshift__")
     def test_count_per_element(self, mock_rrshift):
-        mock_pcollection, ops = self._test_helper()
-        ops.count_per_element(mock_pcollection, "stage_name")
+        mock_pcollection, backend = self._test_helper()
+        backend.count_per_element(mock_pcollection, "stage_name")
         mock_rrshift.assert_called_once_with("unique_label")
 
     @patch("apache_beam.transforms.ptransform.PTransform.__rrshift__")
     def test_reduce_accumulators_per_key(self, mock_rrshift):
-        mock_pcollection, ops = self._test_helper()
-        ops.reduce_accumulators_per_key(mock_pcollection, "stage_name")
+        mock_pcollection, backend = self._test_helper()
+        backend.reduce_accumulators_per_key(mock_pcollection, "stage_name")
         mock_rrshift.assert_called_once_with("unique_label")
 
-    def test_ops_stage_name_must_be_unique(self):
-        ops_1 = BeamBackend("SAME_OPS_SUFFIX")
-        ops_2 = BeamBackend("SAME_OPS_SUFFIX")
+    def test_backend_stage_name_must_be_unique(self):
+        backend_1 = BeamBackend("SAME_backend_SUFFIX")
+        backend_2 = BeamBackend("SAME_backend_SUFFIX")
         with test_pipeline.TestPipeline() as p:
             col = p | f"UNIQUE_BEAM_CREATE_NAME" >> beam.Create([(6, 1),
                                                                  (6, 2)])
-            ops_1.map(col, lambda x: x, "SAME_MAP_NAME")
+            backend_1.map(col, lambda x: x, "SAME_MAP_NAME")
             with self.assertRaisesRegex(RuntimeError,
                                         expected_regex="A transform with label "
-                                        "\"SAME_MAP_NAME_SAME_OPS_SUFFIX\" "
+                                        "\"SAME_MAP_NAME_SAME_backend_SUFFIX\" "
                                         "already exists in the pipeline"):
-                ops_2.map(col, lambda x: x, "SAME_MAP_NAME")
+                backend_2.map(col, lambda x: x, "SAME_MAP_NAME")
 
     def test_one_suffix_multiple_same_stage_name(self):
-        ops = BeamBackend("UNIQUE_OPS_SUFFIX")
+        backend = BeamBackend("UNIQUE_backend_SUFFIX")
```

Comment:
Should this be in capital letters? (Also in the rest of this file)

Reply:
Thanks!